bundles/org.openhab.binding.satel/src/main/resources/ESH-INF/thing/ethm-1.xml

```diff
@@ -47,6 +47,11 @@
 				<description>Encoding used for all the texts received from the module.</description>
 				<default>windows-1250</default>
 			</parameter>
+			<parameter name="extCommands" type="boolean">
+				<label>Extended commands support</label>
```

Comment:
The first letter of every word in a label should be capitalized.

Reply:
Indeed, fixed.